package.json

```diff
@@ -60,7 +60,9 @@
     "@fortawesome/pro-regular-svg-icons": "^5.15.3",
     "@fortawesome/pro-solid-svg-icons": "^5.15.3",
     "@fortawesome/vue-fontawesome": "^2.0.2",
+    "@vue/composition-api": "^1.0.0-rc.8",
```

Comment:
Will we need it in the apps, too?

Reply:
This lib gonna allow to turn on `noImpliciteThis` flag in the marketplace

Reply:
Yes, if we install this plugin in DS it'll be avaialable in `wnl-platform` and `marketplace-frontend`. Although, it's not recommended to use libraries/plugin that are imported by dependencies.

Summarizing, if you want to use it, you should install it as a direct dependency in your app, otherwise you don't need to know that we installed it.